why-oc.md

```diff
@@ -188,7 +188,7 @@ OpenCore works in the same fashion as any other boot loader, so it respects othe
 
 Where hackintoshing sits is in a legal grey area, mainly that while this is not illegal we are in fact breaking the EULA. The reason this is not illegal:
 
-* We are downloading macOS from [Apple's servers directly](https://github.com/corpnewt/gibMacOS/blob/master/gibMacOS.command#L84)
+* We are downloading macOS from [Apple's servers directly](https://github.com/acidanthera/OpenCorePkg/blob/master/Utilities/macrecovery/macrecovery.py#L125)
```

Comment:
Use a tagged link as it will not break in the future. E.g. https://github.com/acidanthera/OpenCorePkg/blob/0.6.8/Utilities/macrecovery/macrecovery.py#L125

Reply:
Good idea